client/src/util/getLocationString.ts

```diff
@@ -1,4 +1,4 @@
-interface PartialLocation {
+export interface PartialLocation {
```

Comment:
What do you think of linking the type to `Venue`, because it's the same type and same scope.


```ts
import { Venue } from '../generated/graphql';

export type Location = Pick<
  Venue,
  'street_address' | 'city' | 'postal_code' | 'region' | 'country'
>;
```